src/EditorFeatures/Core.Wpf/StringIndentation/StringIndentationTaggerProvider.cs

```diff
@@ -25,120 +23,101 @@
 using Microsoft.VisualStudio.Text.Classification;
 using Microsoft.VisualStudio.Text.Editor;
 using Microsoft.VisualStudio.Text.Tagging;
-using Roslyn.Utilities;
 
-namespace Microsoft.CodeAnalysis.Editor.Implementation.StringIndentation
+namespace Microsoft.CodeAnalysis.Editor.Implementation.StringIndentation;
```

Comment:
I like this form better. Any thoughts to a massive one-time change for this, instead of picking these off piecemeal like this?

Reply:
i'm fine if someone wants to make the massive change :)